assets/dns/daemonset.yaml

```diff
@@ -1,18 +1,10 @@
 kind: DaemonSet
 apiVersion: apps/v1
 metadata:
-  # Name is set at runtime
+  # name and labels are set at runtime
   namespace: openshift-dns
```

Comment:
`desiredDNSDaemonSet` sets the namespace, so it can be removed from the asset.

Reply:
done